app/src/main/res/layout/activity_main.xml

```diff
@@ -2,14 +2,14 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
-    android:id="@+id/ll_Root"
+    android:id="@+id/root_view_layout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:divider="@drawable/separatorh"
+    android:divider="@drawable/separator_horizontal"
     android:orientation="vertical"
     android:showDividers="middle"
-    tools:ignore="NestedWeights"
-    tools:context=".MainActivity">
+    tools:context=".MainActivity"
+    tools:ignore="NestedWeights">
```

Comment:
Android Studio says that you layout is bad :) weight is bad idea but sometime it is ok. Nested weight always bad UI which will leads to lugs and so on. Try to rework it with other type of ViewGroup and this layout will be not so much complicated